schemas/com.snowplowanalytics.snowplow.badrows/generic/jsonschema/1-0-0

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Generic bad row",
+    "self": {
+      "vendor": "com.snowplowanalytics.snowplow.badrows",
+      "name": "generic",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "processor": {
+        "type": "object",
+        "description": "Information about the piece of software responsible for the creation of the bad row",
+        "properties": {
+          "artifact": {
+            "type": "string",
+            "description": "Artifact responsible for the creation of the bad row",
+            "maxLength": 512
+          },
+          "version": {
+            "type": "string",
+            "description": "Version of the artifact responsible for the creation of the bad row",
+            "pattern": "^(\\d+\\.\\d+\\.\\d+.*)$",
+            "maxLength": 32
+          }
+        },
+        "required": [ "artifact", "version" ],
+        "additionalProperties": false
+      },
+      "payload": {
+        "type": "object",
+        "properties": {
+          "event": {
```

Comment:
If it was just `payload` (like in other [unstructured payloads](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.badrows/loader_parsing_error/jsonschema/2-0-0#L12-L15)) string we could use common [`RawPayload`](https://github.com/snowplow-incubator/snowplow-badrows/blob/master/src/main/scala/com.snowplowanalytics.snowplow.badrows/Payload.scala#L49)

Reply:
I actually hesitated to reuse `RawPayload`, I hadn't done it only because of the comment that was saying that this is for payloads received by a collector, but you're right it makes more sense to reuse it and update the comment.

Reply:
Yeah let's fix that "collector" part - some loader bad rows also have it